### PR TITLE
Add models

### PR DESCRIPTION
**Pull request by PrincipleZ**

### CODE REVIEW

File: src/models/post.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+export class Post {
+
+    location: any; 
+    timestamp: any;
+    expiration: any;
```
Comment (PrincipleZ) [inline src/models/post.ts]: timestamp and expiration should be of type Date

File: src/models/post.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    description: string;
+    image: string;
```
Comment (PrincipleZ) [inline src/models/post.ts]: image should be a list of string since we may want user to upload multiple photos

File: src/models/post.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    public constructor (private id: number,
+                        private title: string) {
+
```
Comment (PrincipleZ) [inline src/models/post.ts]: where's the content of the constructor 😢

File: src/models/post.ts (hunk 2 above)
Comment (kangningchen) [reply src/models/post.ts]: makes sense

**Pull request merged**